internal/app/machined/pkg/system/services/containerd.go

```diff
@@ -61,7 +61,7 @@ func (c *Containerd) Runner(config runtime.Configurator) (runner.Runner, error)
 			"--address",
 			constants.ContainerdAddress,
 			"--config",
-			"/etc/containerd.toml",
+			"/etc/containerd/cri.toml",
```

Comment:
The default is `/etc/containerd/config.toml`, so I decided to drop it into `/etc/containerd` and call it "cri" because that is its' role in Talos. It is the containerd used as the CRI. In v0.4 I'd like to change our system services names to `cri` and `containerd`.

Reply:
Does this still make sense when considering the use case for exposing `containerd` configuration options via user data? We've had some requests about adding in things like insecure registries and whatnot to this config. 
I guess the name itself doesn't matter too much, but might be a bit confusing for someone wanting to inspect the configuration ( via osctl ls/read/cp ) and seeing something other than the default.

Reply:
I absolutely think it makes sense. :) It is more confusing IMO to have containerd and system-containerd. We really only use container as the CRI when it comes to Kubernetes.